resources/TrashPandaTheme.icls

```diff
@@ -714,15 +701,15 @@
     <option name="XML_TAG" baseAttributes="DEFAULT_TAG" />
     <option name="XML_TAG_DATA" baseAttributes="TEXT" />
     <option name="XML_TAG_NAME" baseAttributes="DEFAULT_KEYWORD" />
-    <option name="XPATH.KEYWORD" baseAttributes="DEFAULT_KEYWORD" />
+    <option name="XPATH.KEYWORD" baseAttributes="" />
```

Comment:
`XPATH.KEYWORD` should have `baseAttributes="DEFAULT_KEYWORD"` to inherit the base keyword appearance